Tools/AerobridgeTrustedFlight_tools/1_pre_flight_checks_load_operation.py

```diff
@@ -0,0 +1,259 @@
+import asyncio

```

Comment:
needs some comments in the python explaining what it does, also needs to have install script to install needed modules